tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md

```diff
@@ -11,7 +11,7 @@ author_profile: https://github.com/MatKuhr
 ## Prerequisites
  - SAP Cloud Application Programming Model
  - Basic knowledge of Spring Boot and Java (optional)
- - An account on [cloud foundry](https://developers.sap.com/group.scp-1-get-ready.html)
+ - An account on [cloud foundry](group.scp-1-get-ready.html)
```

Comment:
This link does not work. Please make sure that this and all other links are working as expected.

Reply:
https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq1u

Reply:
The linked group is present on production but missing in QA. So what should I do in this case? Revert the change and use the absolute link instead?

Reply:
This particular link has the wrong format. It should be [cloud foundry](group.scp-1-get-ready)